core/embed/models/T3W1/boards/trezor_t3w1_revA.h

```diff
@@ -0,0 +1,101 @@
+#ifndef TREZOR_T3W1_REVA_H_
+#define TREZOR_T3W1_REVA_H_
+
+#define HSE_32MHZ
+#define VDD_1V8 1
+
+#define DISPLAY_COLOR_MODE DMA2D_OUTPUT_RGB565
+#define DISPLAY_LEGACY_HEADER "displays/dsi.h"
+
+#define BACKLIGHT_PWM_FREQ 1000
+#define BACKLIGHT_PWM_TIM TIM17
+#define BACKLIGHT_PWM_TIM_CLK_EN __HAL_RCC_TIM17_CLK_ENABLE
+#define BACKLIGHT_PWM_TIM_AF GPIO_AF14_TIM17
+#define BACKLIGHT_PWM_TIM_OCMODE TIM_OCMODE_PWM1
+#define BACKLIGHT_PWM_TIM_CHANNEL TIM_CHANNEL_1
+#define BACKLIGHT_PWM_TIM_CCR CCR1
+#define BACKLIGHT_PWM_PIN GPIO_PIN_9
+#define BACKLIGHT_PWM_PORT GPIOB
+#define BACKLIGHT_PWM_PORT_CLK_EN __HAL_RCC_GPIOB_CLK_ENABLE
+
+#define NPM1300_I2C_INSTANCE 3
+
+#define I2C_COUNT 4
+#define I2C_INSTANCE_0 I2C3
```

Comment:
How about sorting the I2C indexes like this:

- `I2C_INSTANCE_0` -> `I2C1`
- `I2C_INSTANCE_1` -> `I2C2`
- `I2C_INSTANCE_2` -> `I2C3`
- `I2C_INSTANCE_3` -> `I2C4`